src/components/LevelDiscussion/LevelDiscussion.tsx

```diff
@@ -76,6 +82,19 @@ export default function LevelDiscussion() {
     setRefresh(!refresh);
   }
 
+  async function isLevelCompleted() {
+    const userId = user.uid;
+    const users = await getCollection<User>(Collection.USERS);
+    const userObj = users.find((userItem) => userItem.id == userId);
```

Comment:
There is a function in firebaseUtils that does exactly this thing.
Getting an object by ID, getDocumentById()

Reply:
Okok will add this